src/alerts/service-alerts.test.ts

```diff
@@ -19,17 +22,26 @@ afterEach(() => {
   jest.restoreAllMocks();
 });
 
+type AjaxContract = ReturnType<typeof Ajax>;
+
 describe('getServiceAlerts', () => {
   it('fetches service alerts from GCS', async () => {
+    // Arrange
     const mockGetServiceAlerts = jest.fn().mockReturnValue(Promise.resolve([]));
-    Ajax.mockReturnValue({ FirecloudBucket: { getServiceAlerts: mockGetServiceAlerts } });
+    asMockedFn(Ajax).mockReturnValue({
+      FirecloudBucket: { getServiceAlerts: mockGetServiceAlerts },
+    } as DeepPartial<AjaxContract> as AjaxContract);
 
+    // Act
     await getServiceAlerts();
+
+    // Assert
     expect(mockGetServiceAlerts).toHaveBeenCalled();
   });
 
   it('adds IDs to alerts using hashes of alert content', async () => {
-    Ajax.mockReturnValue({
+    // Assert
```

Comment:
pretty sure you mean // Arrange here

Reply:
Fixed in https://github.com/DataBiosphere/terra-ui/commit/a8f8baaa2703dc40863bbe95c96ef441828a967d.